frontend/src/containers/ResetPassword/ResetPasswordPage.js

```diff
@@ -1,23 +1,30 @@
 import React from 'react';
+
+import querystring from 'querystring';
 import LogInPageHeader from '../../components/LogInPageHeader/LogInPageHeader';
 import NewPassword from '../../components/NewPassword/NewPassword';
 import Footer from '../../components/Footer/Footer';
 import './ResetPasswordPage.css';
 
-const ResetPasswordPage = () => (
-  <div className="resetPasswordPage container-fluid">
-    <div className="row">
-      <LogInPageHeader />
-    </div>
-    <div className="passwordBackgroundImage">
-      <div className="newPasswordCardWrapper">
-        <NewPassword />
-        <div className="loginFooter row">
-          <Footer />
+const ResetPasswordPage = (props) => {
+  const { location } = props;
+  const parsed = querystring.parse(location.search.slice(1));
+  const token = parsed.t;
```

Comment:
@vhoof in your previous feedback you said to use token as the query string. I tried changing line 12 to const token = token and const token = parsed.token. But I got errors with both of those. So I changed it to parsed.t.  Not sure if I understood your feedback on this correctly.